<launch>
	<!-- global rosparam -->
	<param name="use_sim_time" value="true"/>

	<!-- rosbag -->
	<!-- <node pkg="rosbag" type="play" name="player" args="&#45;&#45;clock /home/amsl/ozaki/bagfiles/stick_imu4cam/dynamic/dkan_outdoor/20201024_160325.bag"/> -->
	<node pkg="rosbag" type="play" name="player" args="--clock /home/amsl/ozaki/bagfiles/stick_vlp32imu4cam/with_vicon/20201213_183636.bag"/>

	<!-- rviz -->
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find dnn_attitude_estimation)/rviz_config/camera_real.rviz"/>

	<!-- image -->
	<node pkg="image_transport" type="republish" name="republish_f" args="compressed raw">
		<remap from="in" to="/realsense_f/color/image_raw"/>
		<remap from="out" to="/decompressed_image"/>
	</node>

	<!-- MAIN -->
	<node pkg="dnn_attitude_estimation" name="regression_ekf" type="regression_ekf" output="screen">
		<param name="wait_inipose" type="bool" value="false"/>
		<!-- <param name="use_quaternion_for_rotation" type="bool" value="false"/> -->
		<!-- <param name="observe_imu_acc" type="bool" value="false"/> -->
		<!-- <param name="frame_id" type="string" value="/base_link"/> -->
		<!-- <param name="sigma_ini" type="double" value="1.0e&#45;10"/> -->
		<!-- <param name="sigma_gyro" type="double" value="1.0e&#45;4"/> -->
		<!-- <param name="sigma_acc" type="double" value="1.0e+1"/> -->
		<param name="sigma_dnn" type="double" value="1.0e+1"/>
	</node>

	<!-- evaluation -->
	<!-- <include file="$(find dnn_attitude_estimation)/launch/comparison_with_imu_real.launch"/> -->
	<include file="$(find dnn_attitude_estimation)/launch/comparison_with_gt_real.launch"/>

	<!-- visualization -->
	<include file="$(find dnn_attitude_estimation)/launch/visualize_marker_real.launch"/>
</launch>
